Cargo.toml

```diff
@@ -117,3 +131,4 @@ doc = false
 vendored-openssl = ["openssl/vendored"]
 vendored-libgit2 = ["libgit2-sys/vendored"]
 pretty-env-logger = ["pretty_env_logger"]
+all-static = ['openssl/vendored', 'curl-sys/static-curl', 'curl-sys/force-system-lib-on-osx']
```

Comment:
Who is the target audience of this?  Does rust-lang/rust sometimes enable this, always enable this?  I'm wondering if his should be a default

Reply:
Oof, yeah, thats an interesting trade off.